src/components/MessagesList/MessagesList.vue

```diff
@@ -561,6 +561,8 @@ export default {
 				return t('spreed', 'Today')
 			case 1:
 				return t('spreed', 'Yesterday')
+			case 7:
+				return t('spreed', 'A week ago')
 			default:
 				return t('spreed', '{n} days ago', { n: diffDays })
```

Comment:
```suggestion
 				return n('spreed', .... { n: diffDays })
```
Needs to use the plural function

Don't know out of my head how it is in JS

But actually there should be a relative time function already which you can use directly?

Reply:
There is something for it in moment lib

Reply:
> There is something for it in moment lib

Yeah, but it still needs some customization. We need a combination of `date.fromNow()` and `moment().calendar()` because the first shows the past days correctly ( except for yesterday and today, it shows in hours) and the second shows yesterday and tomorrow ( but along with hh:mm).

Reply:
Or we simply use what the activity app does? https://github.com/nextcloud/activity/blob/master/src/components/ActivityGroup.vue#L32

Reply:
That's still not a default output from the lib. We are not avoiding the hurdle of custom anyhow.

We keep it as simple as it is now :) ? but it's better to use `date.fromNow()` instead of `t('spreed', '{n} days ago', { n: diffDays })` indeed